pkg/types/nutanix/defaults/platform.go

```diff
@@ -5,4 +5,6 @@ import (
 )
 
 // SetPlatformDefaults sets the defaults for the platform.
-func SetPlatformDefaults(p *nutanix.Platform) {}
+func SetPlatformDefaults(p *nutanix.Platform) {
```

Comment:
Unclear implementation. Do we want to use SetPlatformDefaults() to set to default value or to configured value if present?

Reply:
fixed